### PR TITLE
Add upgrade script.

### PR DESCRIPTION
**Pull request by nedSaf**
Adding upgrade script like it says in the server docs.

### CODE REVIEW

**Comment by nader77** [comment]
https://github.com/Gizra/drupal-elm-starter/tree/master/server#upgrade

File: server/upgrade [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+##
+function arguments_usage {
+  TITLE=$(fill_string_spaces "Upgrade $PROFILE_TITLE" 61)
+  USAGE=$(fill_string_spaces "Usage: $0 [options]" 61)
+
+  echo
+  echo -e  "${BGCYAN}                                                                 ${RESTORE}"
+  echo -e "${BGLCYAN}  $TITLE  ${RESTORE}"
+  echo -e  "${BGCYAN}  This will upgrade the Drupal core and Contrib modules.         ${RESTORE}"
+  echo -e  "${BGCYAN}  This will run the database update trough Drush.                ${RESTORE}"
```
Comment (AronNovak) [inline server/upgrade]: "trough" => "through"

File: server/upgrade [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+if [ $DEMO_CONTENT ]; then
+  import_demo_content
+fi
+
+# Run post script (if any).
+run_post_script "upgrade"
+
+
+# Check if we have a working bootstrap.
```
Comment (AronNovak) [inline server/upgrade]: We have it twice, other scripts might need it in the future, this bootstrap check could be a function in the helper scripts.

**Comment by AronNovak** [comment]
@nedSaf It would be nice to return to the tiny comments and make it merged.

**Comment by nedSaf** [comment]
@AronNovak You are right, I totally forgot about this with the other stuff I'm working on, it's rerady now.

**Comment by AronNovak** [comment]
I was just to merge it, but found a tiny issue that can be resolved:
image

```
upgrade: line 14: cd: upgrade: Not a directory
```

**Comment by AronNovak** [comment]
Thanks @nedSaf !

**Pull request merged**